static/monorail-2.0.yaml

```diff
@@ -3766,10 +3837,32 @@ definitions:
           $ref: '#/definitions/tag_rule'
     type: object
 
-  ipmi-obm-service_Obm:
+  nodes_post_obm_by_id:
     description: IPMI OBM settings
     properties:
       config:
+#        properties:
+#          host:
+#            description: BMC address
+#            type: string
+#          password:
+#            description: IPMI password
+#            type: string
+#          user:
+#            description: IPMI username
+#            type: string
+        type: object
+      service:
+        type: string
+    required:
+    - service
+    - config
```

Comment:
@geoff-reid Should `nodeId` be required since it is no longer required by the obms library schemas?